pennylane/devices/tests/test_gates.py

```diff
@@ -41,6 +41,7 @@
     "Identity": qml.Identity(wires=[0]),
     "Snapshot": qml.Snapshot("label"),
     "BasisState": qml.BasisState(np.array([0]), wires=[0]),
+    "BlockEncode": qml.BlockEncode(1, wires=[0]),
```

Comment:
Just a suggestion, maybe this should be a 2x2 matrix so that we are testing a more relevant case.